testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -33,7 +33,6 @@ class DockerComposeNamespace(DockerComposeBaseNamespace):
     BASE_FILES = [
         COMPOSE_FILES_PATH + "/docker-compose.yml",
         COMPOSE_FILES_PATH + "/docker-compose.storage.minio.yml",
-        COMPOSE_FILES_PATH + "/docker-compose.testing.yml",
```

Comment:
This commit I don't understand, can you explain it?

The reason for removing the ports was to allow for parallel execution :shrug:

Reply:
See `extra/integration-testing/docker-compose.yml`. The ports are removed there instead. I had to do this as the services are extracted from `docker-compose config` which expected `docker-compose.testing.yml` to exist. However, this file is generated in a fixture later which in turn caused test_compat to fail.

Reply:
Got it. Thank you.